
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Code-first knowledge base for genetics and brain foundation models and multimodal integration">
      
      
      
        <link rel="canonical" href="https://allison-eunse.github.io/neuro-omics-kb/generated/kb_curated/papers-md/oncology_multimodal_waqas2024/">
      
      
      
      
        
      
      
      <link rel="icon" href="../../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.0">
    
    
      
        <title>Oncology multimodal waqas2024 - Neuro-Omics KB</title>
      
    
    
      <link rel="stylesheet" href="../../../../assets/stylesheets/main.618322db.min.css">
      
        
        <link rel="stylesheet" href="../../../../assets/stylesheets/palette.ab4e12ef.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="light-blue" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#multimodal-data-integration-for-oncology-in-the-era-of-deep-neural-networks-a-review" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../../.." title="Neuro-Omics KB" class="md-header__button md-logo" aria-label="Neuro-Omics KB" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Neuro-Omics KB
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Oncology multimodal waqas2024
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="light-blue" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3zm3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95zm-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="light-blue" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5s-1.65.15-2.39.42zM3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29zm.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14zM20.65 7l-1.77 3.79a7.02 7.02 0 0 0-2.38-4.15zm-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29zM12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/allison-eunse/neuro-omics-kb" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../../../.." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../../../guide/kb_overview/" class="md-tabs__link">
          
  
  
  Guides

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../../../models/genetics/" class="md-tabs__link">
          
  
  
  Models

        </a>
      </li>
    
  

    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../../../integration/" class="md-tabs__link">
          
  
  
  Integration

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../../../data/ukb_data_map/" class="md-tabs__link">
          
  
  
  Data

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../../../code_walkthroughs/" class="md-tabs__link">
          
  
  
  Code walkthroughs

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../../../decisions/2025-11-integration-plan/" class="md-tabs__link">
          
  
  
  Decisions

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


  

<nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../../.." title="Neuro-Omics KB" class="md-nav__button md-logo" aria-label="Neuro-Omics KB" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    Neuro-Omics KB
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/allison-eunse/neuro-omics-kb" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Guides
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Guides
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../guide/kb_overview/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    KB overview
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Models
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Models
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_1" >
        
          
          <label class="md-nav__link" for="__nav_3_1" id="__nav_3_1_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Genetics
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_1">
            <span class="md-nav__icon md-icon"></span>
            
  
    Genetics
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/genetics/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/genetics/caduceus/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Caduceus
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/genetics/dnabert2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    DNABERT-2
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/genetics/evo2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Evo 2
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/genetics/generator/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GENERator
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_2" >
        
          
          <label class="md-nav__link" for="__nav_3_2" id="__nav_3_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Brain
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Brain
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/brain/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/brain/brainlm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BrainLM
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/brain/brainjepa/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Brain-JEPA
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/brain/brainharmony/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Brain Harmony
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/brain/brainmt/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BrainMT
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../models/brain/swift/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SwiFT
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Integration
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Integration
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/integration_strategy/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Strategy
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--draft"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_3" >
        
          
          <label class="md-nav__link" for="__nav_4_3" id="__nav_4_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Analysis recipes
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Analysis recipes
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/analysis_recipes/cca_permutation/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    CCA + permutation
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--ready"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/analysis_recipes/prediction_baselines/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Prediction baselines
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--ready"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/analysis_recipes/partial_correlations/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Partial correlations
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--ready"></span>
  

  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4" >
        
          
          <label class="md-nav__link" for="__nav_4_4" id="__nav_4_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Modality features
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Modality features
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/modality_features/genomics/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Genomics
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--ready"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/modality_features/smri/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    sMRI
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--ready"></span>
  

  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/modality_features/fmri/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    fMRI
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--ready"></span>
  

  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/design_patterns/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Design patterns
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../integration/benchmarks/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Benchmarks
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Data
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    Data
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../data/ukb_data_map/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UKB data map
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../data/schemas/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Schemas
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../data/governance_qc/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Governance & QC
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Code walkthroughs
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            
  
    Code walkthroughs
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Index
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/caduceus_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Caduceus
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/dnabert2_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    DNABERT-2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/evo2_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Evo 2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/generator_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GENERator
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/brainlm_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BrainLM
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/brainjepa_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Brain-JEPA
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/brainharmony_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Brain Harmony
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/brainmt_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BrainMT
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../code_walkthroughs/swift_walkthrough/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SwiFT
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Decisions
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            
  
    Decisions
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../decisions/2025-11-integration-plan/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Integration plan (Nov 2025)
  

    
  </span>
  
  
    
  
  
    <span class="md-status md-status--active"></span>
  

  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  


  
    <a href="https://github.com/allison-eunse/neuro-omics-kb/edit/master/docs/generated/kb_curated/papers-md/oncology_multimodal_waqas2024.md" title="Edit this page" class="md-content__button md-icon" rel="edit">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4zm10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1z"/></svg>
    </a>
  
  


<hr />
<h1 id="multimodal-data-integration-for-oncology-in-the-era-of-deep-neural-networks-a-review">Multimodal Data Integration for Oncology in the Era of Deep Neural Networks: A Review<a class="headerlink" href="#multimodal-data-integration-for-oncology-in-the-era-of-deep-neural-networks-a-review" title="Permanent link">&para;</a></h1>
<p><strong>Authors:</strong> Asim Waqas, Aakash Tripathi, Ravi P. Ramachandran, Paul A. Stewart, Ghulam Rasool<br />
<strong>Year:</strong> 2024<br />
<strong>Venue:</strong> Frontiers in Artificial Intelligence</p>
<hr />
<h2 id="1-classification">1. Classification<a class="headerlink" href="#1-classification" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Domain Category:</strong>  </li>
<li>
<p><strong>Multimodal / Integration</strong> (specifically applied to oncology/cancer research). This is a review paper surveying multimodal deep learning methods—especially Graph Neural Networks (GNNs) and Transformers—for integrating diverse cancer-related data modalities (imaging, omics, clinical records).</p>
</li>
<li>
<p><strong>FM Usage Type:</strong>  </p>
</li>
<li>
<p><strong>Review of multimodal FM and integration methods</strong>. The paper does not propose a new foundation model but systematically reviews how modern deep learning architectures (GNNs, Transformers) are being applied to multimodal oncology data fusion, including references to foundation models like CLIP, GPT-4, and FLAVA.</p>
</li>
<li>
<p><strong>Key Modalities:</strong>  </p>
</li>
<li>Radiological imaging (CT, MRI, PET scans)</li>
<li>Digitized histopathology (whole-slide images, H&amp;E stains)</li>
<li>Multi-omics (genomics, transcriptomics, proteomics, metabolomics)</li>
<li>Electronic health records (EHR) and clinical data</li>
<li>Hybrid/derived modalities (radiomics, pathomics features)</li>
</ul>
<hr />
<h2 id="2-executive-summary">2. Executive Summary<a class="headerlink" href="#2-executive-summary" title="Permanent link">&para;</a></h2>
<p>This review paper surveys the landscape of multimodal data integration in oncology, focusing on how deep neural networks—particularly Graph Neural Networks (GNNs) and Transformers—are being used to fuse diverse cancer data types for improved diagnosis, prognosis, and treatment prediction. Cancer research generates heterogeneous data across multiple scales and modalities: from imaging (radiology, pathology) to molecular profiles (genomics, transcriptomics, proteomics) to clinical records. Traditional unimodal analyses fail to capture the complex, interconnected nature of cancer biology. The authors present a comprehensive taxonomy of multimodal learning approaches, covering fusion strategies (early, intermediate, late), neural architectures (CNNs, RNNs, GNNs, Transformers), and domain-specific applications in oncology. They highlight how GNNs naturally model relationships among heterogeneous entities (patients, genes, images) and how Transformers, through self-attention, can integrate sequences of multimodal tokens. Key studies are reviewed across tasks like tumor classification, survival prediction, treatment response, and biomarker discovery. The paper also discusses major challenges—data heterogeneity, missing modalities, alignment across scales, interpretability, and the need for large labeled datasets—and points to promising directions, including foundation models and self-supervised pretraining. For researchers and clinicians, this review provides both a conceptual framework for understanding multimodal fusion and a practical roadmap for applying state-of-the-art deep learning to personalized cancer care.</p>
<hr />
<h2 id="3-problem-setup-and-motivation">3. Problem Setup and Motivation<a class="headerlink" href="#3-problem-setup-and-motivation" title="Permanent link">&para;</a></h2>
<p><strong>Scientific / practical problem:</strong></p>
<ul>
<li>Cancer is inherently multimodal: its biology spans genomic mutations, protein expression, tissue morphology, organ-level imaging, and clinical phenotypes.</li>
<li><strong>Prediction and personalization goals:</strong></li>
<li>Accurate early diagnosis and cancer subtype classification</li>
<li>Prognosis (survival, recurrence risk)</li>
<li>Treatment response prediction and therapy selection</li>
<li>Discovery of prognostic and predictive biomarkers</li>
<li>Traditional approaches analyze modalities in isolation (e.g., only imaging or only genomics), missing synergistic information that could improve accuracy and clinical utility.</li>
</ul>
<p><strong>Why this is hard:</strong></p>
<ul>
<li><strong>Data heterogeneity:</strong></li>
<li>Different modalities have different dimensionalities, scales, and noise characteristics (e.g., high-resolution images vs sparse genomic variants vs tabular clinical data).</li>
<li>Modalities are collected with different protocols, scanners, and sequencing platforms, leading to batch effects and site-specific biases.</li>
<li><strong>Integration complexity:</strong></li>
<li>No universal representation: images are spatial grids, omics are vectors or graphs, clinical data are tabular.</li>
<li>Determining <em>when</em> and <em>how</em> to fuse (early vs late) requires domain knowledge and empirical tuning.</li>
<li><strong>Missing data:</strong></li>
<li>Not all patients have all modalities (e.g., some lack genomic profiling, others lack certain imaging studies).</li>
<li>Models must handle partial observations gracefully.</li>
<li><strong>Label scarcity and class imbalance:</strong></li>
<li>High-quality multi-modal datasets with expert annotations are rare and expensive.</li>
<li>Many cancer subtypes are rare, leading to imbalanced training sets.</li>
<li><strong>Interpretability and clinical trust:</strong></li>
<li>"Black-box" deep learning predictions are hard to explain, yet clinicians need to understand <em>why</em> a model predicts a certain outcome.</li>
<li>Regulatory and ethical considerations demand transparency.</li>
</ul>
<hr />
<h2 id="4-data-and-modalities">4. Data and Modalities<a class="headerlink" href="#4-data-and-modalities" title="Permanent link">&para;</a></h2>
<p><strong>Oncology data modalities covered in the review:</strong></p>
<ul>
<li><strong>Radiological imaging:</strong></li>
<li>CT, MRI, PET scans</li>
<li>
<p>Used for tumor detection, staging, and monitoring response</p>
</li>
<li>
<p><strong>Digitized histopathology:</strong></p>
</li>
<li>Whole-slide images (WSI) of H&amp;E-stained tissue</li>
<li>Immunohistochemistry (IHC) and other stains</li>
<li>
<p>Pathomics: quantitative features extracted from slides</p>
</li>
<li>
<p><strong>Multi-omics:</strong></p>
</li>
<li><strong>Genomics:</strong> DNA mutations, copy number variations, single nucleotide polymorphisms</li>
<li><strong>Transcriptomics:</strong> RNA-seq, gene expression profiles</li>
<li><strong>Proteomics:</strong> Protein abundance and post-translational modifications</li>
<li><strong>Metabolomics:</strong> Small molecule profiles</li>
<li>
<p><strong>Epigenomics:</strong> DNA methylation, histone modifications</p>
</li>
<li>
<p><strong>Electronic Health Records (EHR) and clinical data:</strong></p>
</li>
<li>
<p>Demographics, clinical notes, laboratory results, treatment history, survival outcomes</p>
</li>
<li>
<p><strong>Radiomics:</strong></p>
</li>
<li>Hand-crafted or learned features from imaging (texture, shape, intensity)</li>
</ul>
<p><strong>Major datasets mentioned:</strong></p>
<ul>
<li><strong>The Cancer Genome Atlas (TCGA):</strong> Pan-cancer multi-omics and clinical data</li>
<li><strong>Genomic Data Commons (GDC):</strong> Centralized repository for TCGA and other NCI programs</li>
<li><strong>UK Biobank, All of Us:</strong> Large-scale cohorts with imaging and genomics</li>
<li><strong>TCIA (The Cancer Imaging Archive):</strong> Radiological imaging datasets</li>
<li>Various cancer-specific cohorts (e.g., NSCLC, breast cancer, glioblastoma)</li>
</ul>
<p><strong>Preprocessing / representation:</strong></p>
<ul>
<li>Images: patches or whole-image embeddings from CNNs</li>
<li>Omics: normalized vectors, sometimes projected into lower dimensions</li>
<li>Clinical: tabular features, often encoded or embedded</li>
<li>Graphs: patients, genes, images as nodes; relationships (co-expression, similarity) as edges</li>
</ul>
<hr />
<h2 id="5-model-foundation-model">5. Model / Foundation Model<a class="headerlink" href="#5-model-foundation-model" title="Permanent link">&para;</a></h2>
<p><strong>Model Types:</strong></p>
<p>The review covers a range of architectures for multimodal fusion:</p>
<table>
<thead>
<tr>
<th>Architecture</th>
<th>Role in Multimodal Oncology</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>CNNs</strong></td>
<td>Feature extraction from images (radiology, pathology)</td>
</tr>
<tr>
<td><strong>RNNs/LSTMs</strong></td>
<td>Sequential clinical data, temporal progression</td>
</tr>
<tr>
<td><strong>Autoencoders/VAEs</strong></td>
<td>Dimensionality reduction, unsupervised feature learning</td>
</tr>
<tr>
<td><strong>GANs</strong></td>
<td>Data augmentation, synthetic image generation</td>
</tr>
<tr>
<td><strong>Graph Neural Networks (GNNs)</strong></td>
<td>Model relationships among patients, genes, and multi-modal entities; handle heterogeneous graphs</td>
</tr>
<tr>
<td><strong>Transformers</strong></td>
<td>Self-attention over multimodal token sequences; pre-trained vision-language models adapted to oncology</td>
</tr>
</tbody>
</table>
<p><strong>Focus on GNNs:</strong></p>
<ul>
<li><strong>Graph representation:</strong></li>
<li>Nodes: patients, genes, images, or feature vectors from different modalities</li>
<li>Edges: similarity (clinical, genomic), co-occurrence, known biological interactions</li>
<li><strong>GNN architectures reviewed:</strong></li>
<li>Graph Convolutional Networks (GCN)</li>
<li>Graph Attention Networks (GAT)</li>
<li>GraphSAGE</li>
<li>Message Passing Neural Networks (MPNN)</li>
<li><strong>Applications:</strong></li>
<li>Patient similarity networks for survival prediction</li>
<li>Gene regulatory networks combined with patient omics</li>
<li>Pathology graphs (cells/patches as nodes) integrated with omics</li>
</ul>
<p><strong>Focus on Transformers:</strong></p>
<ul>
<li><strong>Vanilla Transformers:</strong> Self-attention to integrate sequences of multimodal embeddings.</li>
<li><strong>Vision Transformers (ViT):</strong> Patches of histopathology or radiology images as tokens.</li>
<li><strong>Multimodal Transformers:</strong> </li>
<li>Cross-modal attention between image and text/omics modalities</li>
<li>CLIP-like contrastive learning adapted to radiology-pathology or image-genomics pairs</li>
<li><strong>Foundation models mentioned:</strong> CLIP, GPT-4, FLAVA, and domain-specific models like MedCLIP.</li>
</ul>
<p><strong>Training setup (general patterns):</strong></p>
<ul>
<li><strong>Pretraining:</strong> Often on large unimodal datasets (e.g., ImageNet for images, public omics for gene embeddings), sometimes with self-supervised objectives.</li>
<li><strong>Fine-tuning / transfer learning:</strong> Adapt pretrained encoders to oncology tasks with smaller labeled datasets.</li>
<li><strong>Fusion stages:</strong></li>
<li><strong>Early fusion:</strong> Concatenate raw features before modeling.</li>
<li><strong>Intermediate (joint) fusion:</strong> Learn shared representations mid-network.</li>
<li><strong>Late fusion:</strong> Train separate modality-specific models, combine predictions at the end.</li>
<li><strong>Scale:</strong> Varies widely; some studies use hundreds of patients, others leverage TCGA's thousands of samples. Model sizes range from small task-specific networks to large Transformer-based foundation models.</li>
</ul>
<hr />
<h2 id="6-multimodal-integration-aspects-if-applicable">6. Multimodal / Integration Aspects (If Applicable)<a class="headerlink" href="#6-multimodal-integration-aspects-if-applicable" title="Permanent link">&para;</a></h2>
<p>This is fundamentally a <strong>multimodal integration review</strong>, so this section is central.</p>
<p><strong>Which modalities are integrated:</strong></p>
<ul>
<li>Common pairs/triplets:</li>
<li><strong>Radiology + pathology:</strong> CT/MRI + histopathology WSI</li>
<li><strong>Imaging + omics:</strong> Radiology or pathology + genomics/transcriptomics</li>
<li><strong>Omics + clinical:</strong> Gene expression + EHR/treatment history</li>
<li><strong>Triple integration:</strong> Imaging + omics + clinical (less common, more challenging)</li>
</ul>
<p><strong>How they are integrated:</strong></p>
<ul>
<li><strong>Early fusion:</strong></li>
<li>Concatenate features from all modalities into a single vector and feed into a downstream classifier.</li>
<li><em>Pros:</em> Simple, allows the model to learn joint patterns from the start.</li>
<li>
<p><em>Cons:</em> Can be dominated by the highest-dimensional modality; requires careful normalization; struggles with missing data.</p>
</li>
<li>
<p><strong>Late fusion:</strong></p>
</li>
<li>Train separate models for each modality, then combine predictions (e.g., averaging, voting, stacking).</li>
<li><em>Pros:</em> Preserves modality-specific signals; robust to missing modalities; easier to interpret.</li>
<li>
<p><em>Cons:</em> May miss complex cross-modal interactions.</p>
</li>
<li>
<p><strong>Intermediate (joint) fusion:</strong></p>
</li>
<li>Modality-specific encoders produce embeddings that are fused at a middle layer (e.g., via concatenation, attention, or graph pooling) before final prediction.</li>
<li><em>Pros:</em> Balances flexibility and integration.</li>
<li>
<p><em>Cons:</em> Requires architectural design choices; harder to optimize.</p>
</li>
<li>
<p><strong>GNN-based fusion:</strong></p>
</li>
<li>Construct a heterogeneous graph with nodes from different modalities (patient omics, image features, clinical variables).</li>
<li>GNN message passing aggregates cross-modal information.</li>
<li>
<p><em>Example:</em> A patient node connected to its gene expression profile node and its pathology image embedding node; GNN learns to propagate and combine information.</p>
</li>
<li>
<p><strong>Transformer-based fusion:</strong></p>
</li>
<li>Represent each modality as a sequence of tokens (e.g., image patches, genomic regions, clinical features).</li>
<li>Self-attention and cross-attention layers integrate across modalities.</li>
<li><em>Example:</em> Multimodal Transformer taking pathology image patches and omics embeddings as separate token sets, with attention heads learning cross-modal dependencies.</li>
</ul>
<p><strong>Why this integration is useful:</strong></p>
<ul>
<li><strong>Complementary information:</strong> Imaging reveals spatial tumor characteristics, omics show molecular drivers, clinical data provide context (age, stage, treatment).</li>
<li><strong>Improved prediction:</strong> Studies show multimodal models often outperform unimodal baselines on survival, classification, and treatment response tasks.</li>
<li><strong>Biological insight:</strong> Cross-modal associations (e.g., imaging phenotypes correlated with gene expression) can reveal biomarkers and mechanisms.</li>
<li><strong>Personalization:</strong> Comprehensive profiles enable tailored treatment recommendations.</li>
</ul>
<p><strong>Relation to the integration baseline plan:</strong></p>
<ul>
<li><strong>Late fusion first under heterogeneous semantics:</strong></li>
<li>The review aligns with this principle: many successful oncology studies use late fusion or ensemble methods, preserving modality-specific encoders.</li>
<li>
<p>GNNs and Transformers can implement late fusion naturally (separate encoding + graph/attention-based aggregation).</p>
</li>
<li>
<p><strong>Robustness and evaluation discipline:</strong></p>
</li>
<li>The review emphasizes the need for rigorous cross-validation, proper train/test splits (especially important given small sample sizes), and metrics like AUROC, AUPRC, C-index for survival.</li>
<li>
<p>Challenges of missing data and distribution shift are highlighted, aligning with the baseline plan's focus on residualization, covariate adjustment, and bootstrap confidence intervals.</p>
</li>
<li>
<p><strong>CCA and permutation testing:</strong></p>
</li>
<li>
<p>Not explicitly covered in the review, but the principle of exploring modality correlations before heavy fusion is implicit in studies that perform feature selection or canonical correlation analysis on omics and imaging features.</p>
</li>
<li>
<p><strong>Modality sequencing:</strong></p>
</li>
<li>The review suggests starting with well-characterized modalities (e.g., standard imaging + omics) and progressively adding more complex ones (e.g., pathology WSI, radiomics), consistent with the plan's incremental approach.</li>
</ul>
<hr />
<h2 id="7-experiments-and-results">7. Experiments and Results<a class="headerlink" href="#7-experiments-and-results" title="Permanent link">&para;</a></h2>
<p><strong>Tasks / benchmarks reviewed:</strong></p>
<p>The paper surveys a wide range of studies across multiple oncology tasks:</p>
<ul>
<li><strong>Tumor classification and subtyping:</strong></li>
<li>GNN and Transformer models classify cancer types (e.g., glioma grades, breast cancer molecular subtypes) using combined imaging and omics.</li>
<li>
<p>Studies report accuracy improvements of 3-10% over unimodal baselines.</p>
</li>
<li>
<p><strong>Survival prediction and prognosis:</strong></p>
</li>
<li>Multimodal Cox regression models, GNN-based survival networks, and attention-based models integrate clinical, omics, and imaging data.</li>
<li>
<p>C-index improvements of ~0.05-0.15 compared to clinical-only or omics-only models.</p>
</li>
<li>
<p><strong>Treatment response prediction:</strong></p>
</li>
<li>Predicting response to chemotherapy, immunotherapy, or targeted therapies.</li>
<li>
<p>Multimodal approaches combining radiology (baseline tumor imaging) with genomics (mutation profiles) show better discrimination (AUC gains of 0.05-0.10).</p>
</li>
<li>
<p><strong>Biomarker discovery:</strong></p>
</li>
<li>GNNs identify gene modules and image features associated with outcomes.</li>
<li>Transformers' attention weights highlight cross-modal associations (e.g., specific image regions correlating with gene expression patterns).</li>
</ul>
<p><strong>Baselines:</strong></p>
<ul>
<li>Unimodal models (imaging-only, omics-only, clinical-only)</li>
<li>Traditional ML methods (logistic regression, random forests on concatenated features)</li>
<li>Early fusion baselines (simple concatenation + MLP)</li>
</ul>
<p><strong>Key findings (trends and insights):</strong></p>
<ul>
<li><strong>Multimodal consistently beats unimodal:</strong> Across most studies, integrating multiple data types improves predictive performance, often significantly.</li>
<li><strong>Fusion strategy matters:</strong> Late fusion and intermediate fusion tend to outperform early fusion, especially when modalities have very different characteristics.</li>
<li><strong>GNNs excel at relational data:</strong> When patient or gene relationships are explicitly modeled, GNNs capture network effects that simpler models miss.</li>
<li><strong>Transformers scale well:</strong> Transformer-based models benefit from larger datasets and can leverage pretrained vision-language models (transfer learning from CLIP-like architectures).</li>
<li><strong>Interpretability via attention:</strong> Attention weights in Transformers and GNN message passing provide some interpretability, highlighting which modality or feature drives predictions.</li>
<li><strong>Challenges remain:</strong> Performance gains are modest in some cases; missing data and small sample sizes limit generalization; computational cost is high for large-scale models.</li>
</ul>
<hr />
<h2 id="8-strengths-limitations-and-open-questions">8. Strengths, Limitations, and Open Questions<a class="headerlink" href="#8-strengths-limitations-and-open-questions" title="Permanent link">&para;</a></h2>
<p><strong>Strengths:</strong></p>
<ul>
<li><strong>Comprehensive survey:</strong> Covers a wide range of architectures (CNNs, RNNs, GANs, GNNs, Transformers) and applications in oncology.</li>
<li><strong>Taxonomy and framework:</strong> Provides a clear taxonomy of fusion strategies and multimodal learning paradigms, useful for researchers entering the field.</li>
<li><strong>Focus on emerging methods:</strong> Highlights GNNs and Transformers, which are underexplored in oncology compared to computer vision and NLP.</li>
<li><strong>Identifies data resources:</strong> Lists major multimodal oncology datasets (TCGA, GDC, TCIA, UK Biobank), facilitating reproducible research.</li>
<li><strong>Balances technical depth and accessibility:</strong> Suitable for both ML researchers new to oncology and oncology researchers new to advanced deep learning.</li>
</ul>
<p><strong>Limitations:</strong></p>
<ul>
<li><strong>Limited discussion of causal inference:</strong> The review focuses on predictive modeling; less attention to causal relationships or confounding (e.g., how to distinguish direct biological effects from correlations).</li>
<li><strong>Sparse coverage of robustness and fairness:</strong> Issues like model bias across demographics, generalization to external cohorts, and adversarial robustness are mentioned but not deeply explored.</li>
<li><strong>Lack of standardized benchmarks:</strong> The field lacks common evaluation protocols and public leaderboards, making it hard to compare methods across studies.</li>
<li><strong>Interpretability still nascent:</strong> While attention weights and GNN message passing offer some transparency, true mechanistic interpretability (linking predictions to biological pathways) remains an open challenge.</li>
<li><strong>Computational and data barriers:</strong> Many proposed methods require large computational resources and extensive labeled data, limiting accessibility for smaller research groups and clinical settings.</li>
</ul>
<p><strong>Open Questions and Future Directions:</strong></p>
<ul>
<li><strong>Foundation models for oncology multimodal data:</strong></li>
<li>
<p>Can we pretrain large multimodal Transformers on diverse cancer datasets (like CLIP for vision-language) to create a general-purpose oncology FM that transfers to many downstream tasks?</p>
</li>
<li>
<p><strong>Handling missing modalities robustly:</strong></p>
</li>
<li>
<p>Develop architectures that gracefully handle partial observations (e.g., modality dropout during training, imputation via cross-modal generation).</p>
</li>
<li>
<p><strong>Causal multimodal modeling:</strong></p>
</li>
<li>
<p>Move beyond association to causal discovery: which modality changes <em>drive</em> outcomes? How to design experiments or observational studies to infer causality?</p>
</li>
<li>
<p><strong>Fairness and generalization:</strong></p>
</li>
<li>
<p>Ensure multimodal models perform equitably across different patient demographics, cancer subtypes, and institutions (multi-site validation, fairness-aware training).</p>
</li>
<li>
<p><strong>Integration with clinical workflows:</strong></p>
</li>
<li>
<p>Design models that output actionable, interpretable predictions usable by oncologists in real-time decision-making.</p>
</li>
<li>
<p><strong>Self-supervised and few-shot learning:</strong></p>
</li>
<li>
<p>Leverage unlabeled multimodal data (vast amounts available) via self-supervised pretraining; adapt models to rare cancers with few labeled examples using few-shot learning.</p>
</li>
<li>
<p><strong>Explainability and biological insight:</strong></p>
</li>
<li>Develop methods to extract mechanistic understanding from multimodal models (e.g., which genes and image features co-vary and why?).</li>
</ul>
<hr />
<h2 id="9-context-and-broader-impact">9. Context and Broader Impact<a class="headerlink" href="#9-context-and-broader-impact" title="Permanent link">&para;</a></h2>
<p><strong>Position in the FM and multimodal learning landscape:</strong></p>
<ul>
<li>This review sits at the intersection of <strong>multimodal machine learning</strong> and <strong>biomedical AI</strong>, specifically focused on cancer.</li>
<li>It relates to broader trends in foundation models:</li>
<li><strong>CLIP, GPT-4, FLAVA</strong> demonstrate that large-scale pretraining on multimodal data (vision + language) yields versatile representations.</li>
<li>
<p>Oncology is following suit: researchers are exploring whether similar pretraining strategies (e.g., on large radiology-pathology-omics datasets) can yield "cancer foundation models."</p>
</li>
<li>
<p><strong>Analogy to well-known ideas:</strong></p>
</li>
<li>GNNs for multimodal oncology are like "knowledge graphs for cancer," where nodes and edges capture heterogeneous entities and relationships.</li>
<li>Transformers for multimodal oncology are like "BERT/GPT but for diverse cancer data tokens," using self-attention to integrate across modalities.</li>
</ul>
<p><strong>Relation to the integration baseline plan:</strong></p>
<ul>
<li>The review's taxonomy (early vs late fusion, GNN vs Transformer architectures) directly informs the baseline plan's integration strategy recommendations.</li>
<li><strong>Late fusion</strong> (modality-specific encoders + final aggregation) is a recurring theme in successful studies, consistent with the plan's preference for preserving modality-specific signals.</li>
<li>The emphasis on <strong>evaluation rigor</strong> (cross-validation, proper metrics, missing data handling) aligns with the plan's robustness and evaluation discipline.</li>
<li>The review highlights <strong>GNNs and Transformers</strong> as promising architectures for escalation beyond simple concatenation-based fusion, matching the plan's suggestion to explore two-tower contrastive or hub-token architectures if late fusion proves valuable.</li>
</ul>
<p><strong>Why this paper is a useful reference:</strong></p>
<ul>
<li><strong>Educational value:</strong> For a new grad student, this review provides a structured entry point into multimodal oncology, with clear definitions, examples, and a roadmap of key papers.</li>
<li><strong>Design patterns:</strong> The taxonomy of fusion strategies and architectures serves as a design template for building new multimodal systems in cancer or other biomedical domains.</li>
<li><strong>Data resources:</strong> The compilation of datasets accelerates research by pointing to readily available multimodal cohorts.</li>
<li><strong>Future directions:</strong> The open questions guide thesis topics and grant proposals, highlighting high-impact areas for methodological development.</li>
</ul>
<hr />
<h2 id="10-key-takeaways-bullet-summary">10. Key Takeaways (Bullet Summary)<a class="headerlink" href="#10-key-takeaways-bullet-summary" title="Permanent link">&para;</a></h2>
<p><strong>Problem:</strong></p>
<ul>
<li>Cancer data is inherently multimodal (imaging, omics, clinical records), but traditional analyses treat modalities in isolation.</li>
<li>Integrating diverse data types promises improved diagnosis, prognosis, and treatment prediction, but poses significant technical and domain challenges.</li>
</ul>
<p><strong>Method / Model:</strong></p>
<ul>
<li>The review surveys <strong>deep neural network architectures</strong> for multimodal fusion in oncology:</li>
<li><strong>CNNs and RNNs</strong> for imaging and sequential data</li>
<li><strong>Graph Neural Networks (GNNs)</strong> for modeling relationships among patients, genes, and features across modalities</li>
<li><strong>Transformers</strong> for self-attention-based integration of multimodal token sequences</li>
<li><strong>Fusion strategies</strong> covered:</li>
<li><strong>Early fusion:</strong> Concatenate features before modeling</li>
<li><strong>Late fusion:</strong> Separate modality-specific models, combine predictions</li>
<li><strong>Intermediate fusion:</strong> Joint representations learned mid-network, often via attention or graph pooling</li>
<li>The review emphasizes <strong>foundation model</strong> concepts (pretraining, transfer learning) and references CLIP, GPT-4, FLAVA as inspiration for oncology-specific multimodal FMs.</li>
</ul>
<p><strong>Results / Insights:</strong></p>
<ul>
<li>Multimodal models generally <strong>outperform unimodal baselines</strong> on classification, survival, and treatment response tasks, with typical improvements of 3-15% in accuracy/AUC and 0.05-0.15 in C-index.</li>
<li><strong>Late and intermediate fusion</strong> strategies often yield the best performance, preserving modality-specific information while capturing cross-modal interactions.</li>
<li><strong>GNNs</strong> are particularly effective when relational structure (patient networks, gene-gene interactions) is explicitly modeled.</li>
<li><strong>Transformers</strong> scale well and benefit from pretrained vision-language models, showing promise for large-scale oncology FMs.</li>
<li><strong>Interpretability</strong> is improved via attention weights and GNN message passing, but mechanistic understanding remains limited.</li>
</ul>
<p><strong>Why it matters:</strong></p>
<ul>
<li>This review provides a <strong>comprehensive, structured overview</strong> of multimodal deep learning in oncology, filling a gap in the literature by focusing on GNNs and Transformers.</li>
<li>It offers a <strong>taxonomy and design framework</strong> that researchers can use to build and evaluate multimodal systems.</li>
<li>By highlighting <strong>data resources, challenges, and future directions</strong>, it accelerates progress toward personalized cancer care powered by integrated, interpretable AI.</li>
<li>For the <strong>integration baseline plan</strong>, this review validates key principles (late fusion under heterogeneity, robustness discipline, modality sequencing) and points to GNNs/Transformers as architectural choices for escalation beyond simple baselines.</li>
</ul>
<hr />












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../../../..", "features": ["navigation.tabs", "navigation.sections", "navigation.instant", "navigation.top", "content.code.copy", "content.tabs.link", "toc.integrate", "toc.follow", "content.action.edit"], "search": "../../../../assets/javascripts/workers/search.7a47a382.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../../../assets/javascripts/bundle.e71a0d61.min.js"></script>
      
    
  </body>
</html>